arch/arm/src/stm32f7/stm32_lse.c

```diff
@@ -1,9 +1,9 @@
 /****************************************************************************
  * arch/arm/src/stm32f7/stm32_lse.c
  *
- *   Copyright (C) 2017 Gregory Nutt. All rights reserved.
+ *   Copyright (C) 2017, 2021 Gregory Nutt. All rights reserved.
  *   Authors: Gregory Nutt <gnutt@nuttx.org>
- *            David Sidrane <david_s5@nscdg.com>
+ *            David Sidrane <david.sidrane@nscdg.com>
```

Comment:
Could you please change this file license to Apache? You and Greg already assigned the ICLA so no reason to keep it as BSD

Reply:
We can in a separate PR

Reply:
It is a good practice to submit the patches with the coding style fixes and license changes, this is the way we are doing it. It doesn't need to be a new PR, you can just add a new commit to this PR.

Reply:
Yes but It is a critical code change and all the files in F7/HS need the license changed. That should be a pass with NO code changes.  .... You will see this in the next PR

Reply:
Ok

Reply:
Have a look at https://github.com/apache/incubator-nuttx/pull/2945

Reply:
@acassis - Was your OK to merger this?